db/db_local/install.go

```diff
@@ -76,8 +76,17 @@ func EnsureDBInstalled(ctx context.Context) (err error) {
 		return fmt.Errorf("Download & install steampipe-postgres-fdw... FAILED!")
 	}
 
+	statushooks.SetStatus(ctx, "Preparing backups...")
+	var dbName *string
```

Comment:
this is a bit magic - prepareBackup returns the database name?

If we have to do it this way we need a comment making this clear

Reply:
// prepareBackup returns the existing database name - we use this when creating the new database